ambuda/views/proofing/user.py

```diff
@@ -53,9 +53,13 @@ def activity(username):
     session = q.get_session()
     recent_revisions = (
         session.query(db.Revision)
-        .options(orm.defer(db.Revision.content))
+        .options(
+            orm.defer(db.Revision.content),
+            orm.joinedload(db.Revision.page).load_only(db.Page.id, db.Page.slug),
```

Comment:
Can we add a comment explaining that we only fetch relevant content? That we
1. Skip fetching Revision content and
2. Only fetch the `ID` and the `slug` of the attached page

Reply:
These comments would just be describing the query. My view is that comments should explain the "why" instead of the "what," and here the "what" is clearly to avoid loading certain fields. Even "drop certain fields to optimize the query" is redundant given that it's implicit in the fact that we're using a more complex query form here.